nfd-daemonset-combined.yaml.template

```diff
@@ -141,7 +141,8 @@ metadata:
   name: nfd-worker-conf
   namespace: node-feature-discovery
 data:
-  nfd-worker.conf: | ### <NFD-WORKER-CONF-START-DO-NOT-REMOVE>
+  nfd-worker.conf: |
+    ### <NFD-WORKER-CONF-START-DO-NOT-REMOVE>
```

Comment:
Please don't edit the templates. I think it's nicer that these comments do not end up in the actual config map that gets created

Reply:
Such formatting was generated by 'make templates' execution. should I fix it?

Reply:
OK, sorry 😬  I'll check that, don't change it

Reply:
Please, ignore my comment above. It was an issue on my environment

Reply:
Hmm, I can't possibly see how this could happen by the sed script 🤔  What OS and sed version are you using? Are you using some kind of automatic YAML formatter tool?

`values.yaml` did not get automatically mangled like this. Please change these back as they were.